script.js

```diff
@@ -4,16 +4,18 @@
 // be 1. If the user gives the numbers 3 and 2, the result should still
 // be 1.
 //
-// Hint: To get the user input and store it in a variable, we can do 
+// Hint: To get the user input and store it in a variable, we can do
 // `var someVar = prompt("Give me a high five");`. Consider asking for 1
 // number at a time first.
 // ====================================================================
 // Write your solution below.
 // ====================================================================
 
+var userNumOne = parseInt(prompt("Provide the first number", "0"));
+var userNumTwo = parseInt(prompt("Provide the second number", "0"));
 
-
-
+var result = userNumOne - userNumTwo;
+console.log(result);
```

Comment:
But this gives -1 if userNumOne is 1 and userNumTwo is 2, when the difference should be 1... Maybe a if-else statement is needed here to carry out an extra step if the initial result is negative?